library/src/blas_ex/rocblas_gemm_ex.hpp

```diff
@@ -66,13 +66,13 @@ static rocblas_status device_strided_batched_matrix_copy(const To*      src,
     To *dataD, const To *dataC, const Ti *dataA, const Ti *dataB, Tc alpha, Tc beta,              \
         size_t strideD1J, size_t strideD2K, size_t strideC1J, size_t strideC2K, size_t strideA1L, \
         size_t strideA2K, size_t strideB1J, size_t strideB2K, size_t sizeI, size_t sizeJ,         \
-        size_t sizeK, size_t sizeL, hipStream_t stream, size_t numInputEvents, void *dummy1,      \
-        void *dummy2
+        size_t sizeK, size_t sizeL, hipStream_t stream, size_t numInputEvents,                    \
+        hipEvent_t *startEvent, hipEvent_t *stopEvent
 
```

Comment:
This is only used in the old Tensile client. Do you intend to add this to both the old and new client?

Reply:
I already had the changes made for the old client... indifferent how we go about it.

Reply:
> I already had the changes made for the old client... indifferent how we go about it.

As long as it passes CI when built with old client (`install.sh -r`), I'm fine with it.